pkgs/tools/misc/bonsai/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchFromSourcehut
+, gitUpdater
+, hare
+, hare-ev
+, hare-json
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bonsai";
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~stacyharper";
+    repo = "bonsai";
+    rev = "v${version}";
+    hash = "sha256-jOtFUpl2/Aa7f8JMZf6g63ayFOi+Ci+i7Ac63k63znc=";
+  };
+
+  nativeBuildInputs = [
+    hare
+  ];
+
+  buildInputs = [
+    hare-ev
+    hare-json
+  ];
+
+  preConfigure = ''
+    export HARECACHE=$(mktemp -d)
+  '';
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  doCheck = true;
+
+  passthru.updateScript = gitUpdater {
+    rev-prefix = "v";
+  };
+
+  meta = with lib; {
+    description = "Bonsai is a Finite State Machine structured as a tree";
```

```suggestion
    description = "Finite State Machine structured as a tree";
```
`meta.description` should not repeat the package name.